### DISCUSSION TITLE
Sometimes app doesn't have a tray icon

### DISCUSSION THREAD

**Discussion by TheBestPessimist** [seerge/g-helper#586] [Q&A]
![image](https://github.com/seerge/g-helper/assets/4482210/90f5590c-27bc-4d49-aa3b-73897784c247)

no idea how to repro unfortunately :(.
Is this a bug or a feature?

**Comment by seerge** [comment]
@TheBestPessimist may be you just have too many icons in windows tray, so they don't "fit" ? or windows starts hiding them ? try to lower that number

**Reply by TheBestPessimist** [reply]
i definitely dont have too many. those in the screenshot are all of them. Moreover, during worktime i have almost twice as many, and ghelper is still displayed. actually, it's just ghelper which disappears sometimes, none of the other apps' icons disappear.

in this case, i honestly think something is up with ghelper.

**Reply by seerge** [reply]
Well, try find some scenario when it disappears:) or just run app by yourself from visual studio and check if tray icon is in place:)

**Comment by TheBestPessimist** [comment]
Related:
- https://github.com/seerge/g-helper/issues/798 - duplicate by me
- https://github.com/seerge/g-helper/issues/668 - added a manual 1 sec delay @ startup
